sdk/eventhubs/assets.json

```
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "rust",
+  "Tag": "rust/azure_messaging_eventhubs_checkpointstore_blob_e9c892cb82",
+  "TagPrefix": "rust/azure_messaging_eventhubs_checkpointstore_blob"
```

Since you moved it, use the service directory name. Might need to move the files and re-push in your `.assets/???` directory as well.

```suggestion
  "TagPrefix": "rust/eventhubs"
```